lib/shared.bash

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+function strip_quotes() {
+  echo "${1}" | sed "s/^[ \t]*//g;s/[ \t]*$//g;s/[\"']//g"
+}
+
+# AWS SM is only in very recent AWS CLI versions, and isn't on Amazon Linux 2 AMIs (as of July 2018)
+docker pull infrastructureascode/aws-cli
+
+function get_secret_value() {
+  local secretId="$1"
+
+  # Extract the secret string and secret binary
+  local secrets=$(docker run \
```

Comment:
I'll update this to remove the subshell invoke, and thus errors will propagate correctly. It also means I can expand the tests with bats mock.